chore: removed some doc files and put them in the community repo

pratik2315 opened the pull request:
**Description**

- removed the doc files and put them in the asyncapi/community repo

derberg reviewed (dismissed):
Thanks @pratik2315 !!!

related to https://github.com/asyncapi/community/pull/319

derberg commented:
/rtm

derberg commented:
@pratik2315 can you update https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md with new links pointing to community repo? also please check if other documents from this repo need an update

pratik2315 commented:
Hey, so i have updated the the CONTRIBUTING.md file with links to the community repo. Please check it out! I also checked the other docs to see if they needed any updates but I couldn't see any which really needed any major update😅 Thanks.

alequetzalli commented on CONTRIBUTING.md:
```diff
@@ -72,8 +72,11 @@ Prefix that follows specification is not enough though. Remember that the title
 Happy contributing :heart:
 
+## Community
+AsyncAPI Community welcomes you! You can Star⭐ our [Community](https://github.com/asyncapi/community) repository to get the latest updates about Community Meetings, Youtube Sessions, Mailing lists, Slack channel and much more! Make sure to familiarize yourself with our [GitHub workflow](https://github.com/asyncapi/community/blob/master/git-workflow.md) and don't forget to follow [Slack Etiquette' ...
```
```suggestion
Welcome to the AsyncAPI Community! You can Star ⭐ our [Community repository](https://github.com/asyncapi/community) to get the latest updates about Community Meetings, Youtube Sessions, Mailing lists, Slack channels, and much more! Make sure to familiarize yourself with our [GitHub workflow](https://github.com/asyncapi/community/blob/master/git-workflow.md), and don't forget to follow our [Slack Etiquette](https://github.com/asyncapi/community/blob/master/slack-etiquette.md) while interacting with community members!
```

alequetzalli reviewed (changes requested):
@pratik2315 Just a tiny feedback item 🎉

pratik2315 commented:
All done! Can u take a look @alequetzalli ?

alequetzalli reviewed (approved):
🚢 🚢 🚢 🚢

derberg commented on CONTRIBUTING.md:
```diff
@@ -3,7 +3,7 @@ We love your input! We want to make contributing to this project as easy and tra
 ## Contribution recogniton
 
-We use [All Contributors](https://allcontributors.org/docs/en/specification) specification to handle recognitions. For more details read [this](recognize-contributors.md) document. 
+We use [All Contributors](https://allcontributors.org/docs/en/specification) specification to handle recognitions. For more details read [this](recognize-contributors.md) document.
```
I think `[this](recognize-contributors.md)` needs update too

derberg commented on CONTRIBUTING.md:
```diff
@@ -51,7 +51,7 @@ Please use our issues templates that provide you with hints on what information
 ## Pull Requests
 
-**Please, make sure you open an issue before starting with a Pull Request, unless it's a typo or a really obvious error.** Pull requests are the best way to propose changes to the specification. Get familiar with our document that explains [Git workflow](git-workflow.md) used in our repositories. 
+**Please, make sure you open an issue before starting with a Pull Request, unless it's a typo or a really obvious error.** Pull requests are the best way to propose changes to the specification. Get familiar with our document that explains [Git workflow](git-workflow.md) used in our repositories.
```
I think `[Git workflow](git-workflow.md)` needs update too

derberg commented on CONTRIBUTING.md:
```diff
@@ -72,8 +72,11 @@ Prefix that follows specification is not enough though. Remember that the title
 Happy contributing :heart:
 
+## Community
+Welcome to the AsyncAPI Community! You can Star ⭐ our [Community repository](https://github.com/asyncapi/community) to get the latest updates about Community Meetings, Youtube Sessions, Mailing lists, Slack channels and much more! Make sure to familiarize yourself with our [GitHub workflow](https://github.com/asyncapi/community/blob/master/git-workflow.md), and don't forget to follow our [Slack E ...
```
what do your mean that you can star a repo and as a result get latest updates about different AsyncAPI things?

Also, git-workflow is already mentioned in another section.

Regarding slack etiquette, if we really want to link it in this document, it should go to `Issues` section where we mention Slack.

And in general, what is the point of this section in the contribution guide as current information is not much helpful. If we think this section is needed, I think we should just link to community repo and this issue https://github.com/asyncapi/community/issues/1

We have a lot of info here already https://github.com/asyncapi/ 🤔

pratik2315 replied:
Ohh! got it! I looked at the said sections and most of the info is already mentioned there! so I guess instead of creating diff section for community I'll just move slack etiquette to the issues section and update the git-workflow link which is already there and point it to community👍

derberg reviewed (changes requested):
There are some things to fix.

Also please:
- fix also `[read the full text](./CODE_OF_CONDUCT.md)` link in CONTRIBUTING.md
- you also need to update https://github.com/asyncapi/.github/blob/master/profile/README.md
please make sure that you also update latest links to meetings, as zoom and calendar links are outdated

pratik2315 commented:
> There are some things to fix.
> 
> Also please:
> 
> * fix also `[read the full text](./CODE_OF_CONDUCT.md)` link in CONTRIBUTING.md
> * you also need to update https://github.com/asyncapi/.github/blob/master/profile/README.md
>   please make sure that you also update latest links to meetings, as zoom and calendar links are outdated

I got everything else👍but im confused about the second point here. In the asyncapi/community, there are meeting times and calendar mentioned, are you saying that those are not the updated links? if not where can I find the updated meeting links?

pratik2315 commented:
Hey @derberg so ive added calendar links to the .github README.md file. so those are the only things we should add for now? because the readme file looks kind empty dont u think?😅 thanks for being so patient with me (lemme know if there are any more changes!!)

derberg reviewed (commented):
I'm not sure about this "meetings" section in README.md

as you see for now we were not very detailed on the readme as `.github` repo is rather technical, not "promotional", if you know what I mean 🤔 

we rather need update of outdated meeting-related info here https://github.com/asyncapi/.github/blob/master/profile/README.md

maybe in main readme just link to profile readme?

@alequetzalli what do you think?

pratik2315 commented:
> I'm not sure about this "meetings" section in README.md
> 
> as you see for now we were not very detailed on the readme as `.github` repo is rather technical, not "promotional", if you know what I mean 🤔
> 
> we rather need update of outdated meeting-related info here https://github.com/asyncapi/.github/blob/master/profile/README.md
> 
> maybe in main readme just link to profile readme?
> 
> @alequetzalli what do you think?

Or I could update the meeting links in profile/README.md and use that textfile as our main README file?
I am unable to understand the purpose of profile/README😅 Can't we use it as our main readme file after updation?

derberg commented:
@pratik2315 this readme file is what you can see here https://github.com/asyncapi/ so super important content 😄

pratik2315 commented:
> @pratik2315 this readme file is what you can see here https://github.com/asyncapi/ so super important content 😄

Ohh I see🙌

derberg reviewed (commented):
@pratik2315 once you remove meeting info from main readme and update profile readme with new links, we can merge

derberg commented:
some inspiration here https://github.com/asyncapi/community/pull/336

derberg commented:
@pratik2315 pinging in case you missed it 😉

pratik2315 commented:
Oops! I missed the comments above😅 Ill start on this issue asap🙌

pratik2315 commented:
hey @derberg ! 

- When clicking on the zoom link meeting in profile/readme I was getting redirected to sig meeting waiting room. If I am not wrong those meetings are outdated right? For now, I have removed that particular line and link from profile/readme file
- Added the ics download file line from the community readme which was missing
- kept the main readme file same as before as you instructed

Really sorry for the delay and let me know if i got something wrong😅 thanks!

derberg commented:
- yes, zoom link is outdated, no more promoting zoom link directly
- no joining mailing list
- no need to share ics

best just link to https://www.asyncapi.com/community/meetings. As simple as https://github.com/asyncapi/community/pull/336 that is already approved

no worries about delay mate 🍺

derberg commented on profile/README.md:
```diff
@@ -21,10 +21,7 @@ To get involved with our community, please make sure you are familiar with the p
-    - Join our regular bi-weekly [Zoom meetings](https://zoom.us/j/165106914).
-    - Add our shared [Google Calendar](https://calendar.google.com/calendar?cid=dGJyYmZxNGRlNWJjbmd0OG9rdmV2NGxzdGtAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ). 
-    - Join our [mailing list](https://groups.google.com/forum/#!forum/asyncapi-users).
+- **Join our public meetings**: AsyncAPI hosts different meetings every week. They are focused on different topic, sometimes purely technical and sometimes about community building. Pick one and join us! [Learn more about our meetings.](https://www.asyncapi.com/community/meetings)
```
```suggestion
- **Join our public meetings**: AsyncAPI hosts different meetings every week. They are focused on different topic, sometimes purely technical and sometimes about community building. Pick one and join us! [Learn more about our meetings](https://www.asyncapi.com/community/meetings).
```

derberg commented:
simple suggestion

pratik2315 commented:
> simple suggestion

done!

pratik2315 commented:
> Something bad happened. Many different bot related changes showed up. Force pushing is definitely not recommended 🙂

Extremely sorry, Should I do something??

pratik2315 commented:
Before starting today's work I pulled the latest work from master into my branch maybe that along w/ the rebasing caused some clash?

pratik2315 commented:
Found the error. seems like i rebased 3 commits from other people. 2 from your work and 1 commit from namya's

![Screenshot (71)](https://user-images.githubusercontent.com/77961530/167895388-34e7a488-bbe3-472d-84c3-29bd2bf67327.png)

derberg commented:
tbh, in such cases I'm always to lazy to learn git to the extent that helps to fix it and simply recreate work in fresh branch with new PR 😄

pratik2315 commented:
> tbh, in such cases I'm always to lazy to learn git to the extent that helps to fix it and simply recreate work in fresh branch with new PR 😄

Okay i'll create a new and close this one😁
